### PR TITLE
chore: add alloy db, update maintainers and fmt

### PR DESCRIPTION
**Pull request by bharathkkb**

### CODE REVIEW

**Comment by bharathkkb** [comment]
this has been applied

File: infra/terraform/test-org/org/locals.tf [hunk 1]
```diff
@@ -108,11 +108,11 @@ locals {
       topics      = join(",", [local.common_topics.compute, local.common_topics.containers])
     },
     {
-      name         = "terraform-ecommerce-microservices-on-gke"
```
Comment (apeabody) [inline infra/terraform/test-org/org/locals.tf:111]: We should check why the linter didn't catch originally.

**Review by apeabody — APPROVED** [review]
@bharathkkb LGTM!

File: infra/terraform/test-org/org/locals.tf (hunk 1 above)
Comment (bharathkkb) [reply infra/terraform/test-org/org/locals.tf:111]: It was my bad, I already had applied the changes in this PR locally so I merged the other `ecommerce-microservices-on-gke` PR and applied without checking lint.

Comment (apeabody) [reply infra/terraform/test-org/org/locals.tf:111]: No worries @bharathkkb  :) It was just a reminder to myself to investigate as we've had some linter report pass when they actually failed, just happy that wasn't the situtation.

**Pull request merged**